modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -70,11 +70,13 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 
 		boolean showSuccessPage = ddmFormDisplayContext.isShowSuccessPage();
 
+		boolean expired = DDMFormInstanceStatusUtil.isFormExpired(formInstance, timeZone);
+
 		boolean preview = ddmFormDisplayContext.isPreview();
 		%>
 
 		<c:choose>
-			<c:when test="<%= showSuccessPage || (ddmFormDisplayContext.hasSubmittedAnEntry() && !preview) %>">
+			<c:when test="<%= showSuccessPage || expired || (ddmFormDisplayContext.hasSubmittedAnEntry() && !preview) %>">
```

Comment:
@rodrigopaulino can we order this (`showSuccessPage || expired`) alphabetically?

Reply:
I think we can, because they are supposed to be excluding flags.

Reply:
So I think it's better to sort alphabetically. Thanks!